### ISSUE TITLE
Replace icon and settings picture

### ISSUE THREAD

**Issue by flexits** [flexits/HotCornersWin#9]
A new icon is needed; the current one doesn't look quite suitable for a "hot corners" application. The image in settings must be updated as well.

**Comment by Lisa24Jackson** [comment]
Believe me, I tried many different options for implementing the icon so that it would meet its purpose and not look like a traffic light (although I tried several such options). Hot corners, this isn't an average application where there is some accepted icon template and you can implement something similar. I like the HotFrameFx icon in general, although I modernized it a bit to make it better for me.

![226136941-0b4f49fa-d1b4-4ecc-a3eb-af232f023768](https://github.com/flexits/HotCornersWin/assets/155259522/d92db0d5-5e01-4a35-bdec-2a44cabd86d1)

![hotframefx-logo-fix](https://github.com/flexits/HotCornersWin/assets/155259522/b3b803e1-1984-472e-a162-303440b6ddcb)


But nevertheless, it also seems small in the tray and its purpose is difficult to read. The current version in HotCornersWin doesn't fit the purpose, but at the same time, due to the mosaic tiles, it's not so small and you can distinguish it, although, as I mentioned, this is of course the tiling mode icon, not the hot corners.

More to the point. As a result, I created icons and images for settings in the style of the icon itself, in my opinion, it looks good. 

On:
![icon-new-on](https://github.com/flexits/HotCornersWin/assets/155259522/5ac857d5-8f11-4bbe-85fb-96b7b5678cf5)

Off:
![icon-new-off](https://github.com/flexits/HotCornersWin/assets/155259522/2cdebade-32f5-4604-a05e-bb7d554b4557)

Image for settings:
![display-new](https://github.com/flexits/HotCornersWin/assets/155259522/dcafb86f-6662-4372-bd2d-a1182e576e8b)

I tried to put it on the tray. Yes, it's small (as in all other options and cases), but the essence of the purpose can be caught. In my opinion, it looks not bad and as concise as possible, as I wanted. I really don't know if you'll like it. I'll be grateful for the answer. I'm also attaching the zip archive with the icons and image just in case you want to check.

**Comment by Lisa24Jackson** [comment]
Believe me, I tried many different options for implementing the icon so that it would meet its purpose and not look like a traffic light (although I tried several such options). Hot corners, this isn't an average application where there is some accepted icon template and you can implement something similar. I like the HotFrameFx icon in general, although I modernized it a bit to make it better for me.

![226136941-0b4f49fa-d1b4-4ecc-a3eb-af232f023768](https://github.com/flexits/HotCornersWin/assets/155259522/d92db0d5-5e01-4a35-bdec-2a44cabd86d1)

![hotframefx-logo-fix](https://github.com/flexits/HotCornersWin/assets/155259522/b3b803e1-1984-472e-a162-303440b6ddcb)


But nevertheless, it also seems small in the tray and its purpose is difficult to read. The current version in HotCornersWin doesn't fit the purpose, but at the same time, due to the mosaic tiles, it's not so small and you can distinguish it, although, as I mentioned, this is of course the tiling mode icon, not the hot corners.

More to the point. As a result, I created icons and images for settings in the style of the icon itself, in my opinion, it looks good. 

On:
![icon-new-on](https://github.com/flexits/HotCornersWin/assets/155259522/5ac857d5-8f11-4bbe-85fb-96b7b5678cf5)

Off:
![icon-new-off](https://github.com/flexits/HotCornersWin/assets/155259522/2cdebade-32f5-4604-a05e-bb7d554b4557)

Image for settings:
![display-new](https://github.com/flexits/HotCornersWin/assets/155259522/dcafb86f-6662-4372-bd2d-a1182e576e8b)

I tried to put it on the tray. Yes, it's small (as in all other options and cases), but the essence of the purpose can be caught. In my opinion, it looks not bad and as concise as possible, as I wanted. I really don't know if you'll like it. I'll be grateful for the answer. I'm also attaching the zip archive with the icons and image just in case you want to check.

[icons-display.zip](https://github.com/flexits/HotCornersWin/files/13834148/icons-display.zip)

**Comment by flexits** [comment]
> I really don't know if you'll like it. I'll be grateful for the answer.

Oh that's just wonderful 👍 Thanks a lot!

**Comment by flexits** [comment]
accepted in beta 0.7.0

**Comment by flexits** [comment]
P.S. BTW what software do you use to make icons? I'm not a design guy at all but maybe someday I'll make an effort to learn 😁

**Comment by Lisa24Jackson** [comment]
> P.S. BTW what software do you use to make icons? I'm not a design guy at all but maybe someday I'll make an effort to learn 😁

Well, I'm not a designer either 😁

In this case, I used only Photoshop. But I'd several versions of the cursor icon, at one time I created them, it seems, in Inkscape, when I was using Ubuntu, because they remained with me only in the form of png files.
If there is an opportunity, I use the Adobe package, since it's universal, although it's by subscription.
And from free and open-source, in addition to Inkscape, I can also mention Krita, and of course GIMP if it isn't possible to use Photoshop. There is also Sketch on macOS, I've seen a lot of people using it. I know that [Basic Apple Guy](https://basicappleguy.com/) mainly uses it to create his wallpapers, as well as icons.
